inference-engine/src/mkldnn_plugin/CMakeLists.txt

```diff
@@ -109,7 +109,7 @@ if(GEMM STREQUAL "MKL")
     install(DIRECTORY "${MKL}/include"
             DESTINATION ${IE_CPACK_IE_DIR}/external/mkltiny_lnx
             COMPONENT cpu)
-    install(FILES "${MKLLIB}"
+    install(DIRECTORY "${MKLLIB}"
```

Comment:
looks like the problem is not in `FILES` or `DIRECTORY`, but that `MKLLIB` empty in your case

Reply:
I already tried that's point. (https://github.com/openvinotoolkit/openvino/issues/5965)

It fails too when I explicitly gives $MKL. $MKLLIB variables.

Reply:
Why do you need MKL? This code is removed from current master.

Reply:
I have used MKL our project internally from Vino2020.2,
We decided to update our dependancies including vino to newer versions.
So we need to test 2021.3 ASAP on our environment.